### PR TITLE
cli (ticdc): fix query changefeed does not contain config info and mask sink uri.

### LINKED ISSUE #6362: changefeed query has not filter information
### What did you do?

/ # /cdc cli changefeed query -c redo-apply-cdc-all-node-restart-sync --pd=http://upstream-pd:2379/
{
  "upstream_id": 7122327567255556103,
  "namespace": "default",
  "id": "redo-apply-cdc-all-node-restart-sync",
  "sink_uri": "mysql://root:@downstream.cdc-testbedcl8xv:3306",
  "create_time": "2022-07-20 05:50:23.957",
  "start_ts": 434712405284225025,
  "resolved_ts": 434712435986792452,
  "target_ts": 0,
  "checkpoint_tso": 434712436510818308,
  "checkpoint_time": "2022-07-20 05:52:22.891",
  "sort_engine": "unified",
  "state": "normal",
  "error": null,
  "error_history": null,
  "creator_version": "",
  "task_status": [
    {
      "capture_id": "f7518d4f-dae1-4518-979b-bb7e08d65926",
      "table_ids": [],
      "table_operations": null
    }
  ]
}
/ #

### What did you expect to see?

Pre-6.2.0 version has table filter in query changefeed, it's useful to check filter table.

### What did you see instead?

No any  filter info.

### Versions of the cluster

Upstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):

```console
(paste TiDB cluster version here)
```

Upstream TiKV version (execute `tikv-server --version`):

```console
(paste TiKV version here)
```

TiCDC version (execute `cdc version`):

```console
(paste TiCDC version here)
v6.2.0 nightly
```

### PR DESCRIPTION
**Pull request by asddongmen**
### What problem does this PR solve?

Issue Number: close #6362

### What is changed and how it works?
- cli changefeed query 
image

- cli changefeed create
image

- cli changefeed update 
image
- cli changefeed remove 
image

#### Tests 

 - Manual test (add detailed scripts or steps below)

### Release note 

```release-note
Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

If you don't think this PR needs a release note then fill it with `None`.
```

### CODE REVIEW

File: cdc/model/http_model.go [hunk 1]
```diff
@@ -140,6 +140,7 @@ type ChangefeedDetail struct {
 	CheckpointTime JSONTime            `json:"checkpoint_time"`
 	Engine         SortEngine          `json:"sort_engine,omitempty"`
 	FeedState      FeedState           `json:"state"`
+	Config         interface{}         `json:"config,omitempty"`
```
Comment (sdojjy) [inline cdc/model/http_model.go:143]: do not use interface

**Review by sdojjy — APPROVED** [review]

**Review by nongfushanquan — APPROVED** [review]

File: pkg/cmd/cli/cli_changefeed_query_test.go [hunk 2]
```diff
@@ -53,15 +60,26 @@ func TestChangefeedQueryCli(t *testing.T) {
...
+	cfV2.EXPECT().GetInfo(gomock.Any(), gomock.Any()).Return(&v2.ChangeFeedInfo{
+		Config: v2.GetDefaultReplicaConfig(),
+	}, nil)
 	os.Args = []string{"query", "--simple=false", "--changefeed-id=bcd"}
+	b := bytes.NewBufferString("")
+	cmd.SetOut(b)
 	require.Nil(t, cmd.Execute())
+	out, err := ioutil.ReadAll(b)
+	require.Nil(t, err)
+	// make suer config is printed
```
Comment (zhaoxinyu) [inline pkg/cmd/cli/cli_changefeed_query_test.go:78]:
```suggestion
	// make sure config is printed
```

File: pkg/util/uri.go [hunk 3]
```diff
@@ -15,7 +15,11 @@ package util
 
 import (
 	"net"
+	"net/url"
 	"strings"
+
+	"github.com/ngaut/log"
```
Comment (zhaoxinyu) [inline pkg/util/uri.go:21]: Maybe it's better to use `github.com/pingcap/log` to keep consistent with other files.

Comment (asddongmen) [reply pkg/util/uri.go:21]: good catch

**Comment by asddongmen** [comment]
God, please help me to merge this PR.

**Pull request merged**